wagtail/admin/templates/wagtailadmin/pages/history.html

```diff
@@ -1,61 +1 @@
-{% load i18n wagtailadmin_tags %}
-
-{% block titletag %}{% blocktrans trimmed with title=subtitle %}Page history for {{ subtitle }}{% endblocktrans %}{% endblock %}
-
-{% block actions %}
-    <a href="{% url 'wagtailadmin_pages:edit' page.id %}" class="button bicolor button--icon">{% icon name="edit" wrapped=1 %}{% trans "Edit this page" %}</a>
-{% endblock %}
-
-{% block results %}
-    {% if object_list %}
-        <table class="listing listing--inline-actions">
-            <thead>
-                <tr>
-                    <th>
-                        {% trans 'Action' %}
-                    </th>
-                    <th>
-                        {% trans 'User' %}
-                    </th>
-                    <th class="updated">
-                        {% trans 'Date / Time' %}
-                    </th>
-                </tr>
-            </thead>
-            <tbody>
-                {% page_permissions page as page_perms %}
-                {% for entry in object_list %}
-                    <tr>
-                        <td>
-                            {% if entry.revision %}<span class="report__results--text">{% endif %}
-                            {{ entry.message }}
-                            {% if entry.revision %}</span>{% endif %}
-                            {% if entry.comment %}
-                                <span class="report__results--comment">{% trans "Comment" %}: <em>{{ entry.comment }}</em></span>
-                            {% endif %}
-                            {% if entry.revision %}
-                                {% if entry.action == 'wagtail.publish' %}
-                                    {% if entry.revision_id == page.live_revision_id %}{% trans 'Live version' as status_label %}{% status status_label classname="w-status--primary" %}{% endif %}
-                                {% elif entry.content_changed %}
-                                    {% if entry.revision == page_latest_revision %}{% trans 'Current draft' as status_label %}{% status status_label classname="w-status--primary" %}{% endif %}
-                                    {% include "wagtailadmin/pages/revisions/_actions.html" with revision=entry.revision %}
-                                {% endif %}
-                            {% endif %}
-                        </td>
-                        <td>
-                            {% include "wagtailadmin/shared/user_avatar.html" with user=entry.user username=entry.user_display_name %}
-                        </td>
-                        <td class="updated">{% human_readable_date entry.timestamp %}</td>
-                    </tr>
-                {% endfor %}
-            </tbody>
-        </table>
-    {% else %}
-        <p>{% trans "No log entries found." %}</p>
-    {% endif %}
-{% endblock %}
-
-{% block no_results %}
-    <p>{% trans "No log entries found." %}</p>
-{% endblock %}
+{% extends 'wagtailadmin/generic/listing.html' %}
```

Comment:
We can also completely remove this file instead, but for now I opted to leave this in, in case people do some template customisations.

Maybe we should take inventory of such files, call them out in the release notes, and remove them in the next major release. Additionally, we could update the view code to still look for `wagtailadmin/pages/history.html` first, before falling back to `wagtailadmin/generic/listing.html`.